package.json

```diff
@@ -47,7 +47,8 @@
     },
     "dependencies": {
         "aws-sdk": "^2.1018.0",
-        "cypress-file-upload": "^5.0.8"
+        "cypress-file-upload": "^5.0.8",
+        "path-browserify": "^1.0.1"
```

Comment:
do you mean to add this dependency?

Reply:
It’s used in the cypress test to pop the end of the URL off, we could probably do something else, my editor was upset about the import. You were also talking about there being a Cypress fucntion to do this but I couldn’t find what you were talking about.